### ISSUE TITLE
Button that opens settings for channel deletion is not keyboard accessible

### ISSUE THREAD

**Issue by kingalg** [TryQuiet/quiet#1593]
Button that opens settings for channel deletion is not keyboard accessible

**Priority** - medium, as it is big a11y problem but it right now it impact only community owner.
**Issue** - it’s not focusable and has no text tor VoiceOver.
**Solution** - make this element focusable and add aria-label=”Channel deletion menu” or something similar.
<img width="520" alt="Screenshot 2023-06-27 at 15 42 19" src="https://github.com/TryQuiet/quiet/assets/68909168/48385452-7e23-4ce3-9e99-65f0906fb974">
